content/news/release-notes.adoc

```diff
@@ -15,6 +15,43 @@ For additional information check our https://www.youtube.com/channel/UCcm2NzDN_U
 
 toc::[]
 
+== 1.26.0
+Sprint Release: October 30th, 2020
+
+Features:
+
+* https://github.com/kiali/kiali/issues/3363[Enable/Disable Mutual TLS by Namespace granularity ]
+* https://github.com/kiali/kiali/issues/3067[Add Date/time (start/stop) filtering for logs tab]
+* https://github.com/kiali/kiali/issues/2232[Improve visualization, filtering and searching in Kiali Logs tab]
+* https://github.com/kiali/kiali/issues/3340[Remove mixer/telemetry-v1 use of _app and _version metric attributes]
+* https://github.com/kiali/kiali/issues/3341[Remove mixer config dependencies]
+* https://github.com/kiali/kiali/issues/3155[Add fullscreen zooming beyond browser chrome]
+* https://github.com/kiali/kiali/issues/3338[(mixer removal) remove use of  istio_request_duration_seconds]
+* https://github.com/kiali/kiali/issues/3303[Support "duration" parameter for pod logs API]
+* https://github.com/kiali/kiali/issues/3315[Short term caching of Prometheus queries in Health Service]
+* https://github.com/kiali/kiali/issues/2235[Improve health check of Istio subcomponents]
+* https://github.com/kiali/kiali/issues/3324[(operator) provide a hidden setting in Kiali CR to turn off operator reconciliation]
+* https://github.com/kiali/kiali/issues/3322[turn off dependent resource watching until we get a better handle on its effects]
```

Comment:
Let's delete this one. This actually was reverted in the next  PR (3324). So dependent resource watching is enabled even in this release.

```suggestion
```